extensions/map-layers/scripts/copy-test-setup.js

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
You may want to make sure this file and the setup-tests.js are excluded from the npm package.